### ISSUE TITLE
Build broken

### ISSUE THREAD

**Issue by nzakas** [eslint/website#919]
The latest build broke:

```
5:56:39 PM: Build ready to start
5:56:41 PM: build-image version: 6ebfb829398e07eb99ad5455777162ff19838822 (xenial)
5:56:41 PM: build-image tag: v3.10.5
5:56:41 PM: buildbot version: 618a04140a44a0a80470bd94a1ccfe6eb588786d
5:56:41 PM: Building without cache
5:56:41 PM: Starting to prepare the repo for build
5:56:41 PM: No cached dependencies found. Cloning fresh repo
5:56:41 PM: git clone https://github.com/eslint/website
5:56:46 PM: Preparing Git Reference refs/heads/master
5:56:47 PM: Parsing package.json dependencies
5:56:48 PM: Starting build script
5:56:48 PM: Installing dependencies
5:56:48 PM: Python version set to 2.7
5:56:50 PM: Downloading and installing node v12.22.10...
5:56:50 PM: Downloading https://nodejs.org/dist/v12.22.10/node-v12.22.10-linux-x64.tar.xz...
5:56:50 PM: Computing checksum with sha256sum
5:56:51 PM: Checksums matched!
5:56:53 PM: Now using node v12.22.10 (npm v6.14.16)
5:56:54 PM: Started restoring cached build plugins
5:56:54 PM: Finished restoring cached build plugins
5:56:54 PM: Attempting ruby version 2.6.2, read from environment
5:56:55 PM: Using ruby version 2.6.2
5:56:56 PM: Using PHP version 5.6
5:56:56 PM: Started restoring cached node modules
5:56:56 PM: Finished restoring cached node modules
5:56:56 PM: Installing NPM modules using NPM version 6.14.16
5:56:59 PM: npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
npm ERR! code EBADPLATFORM
5:56:59 PM: npm ERR! notsup Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
5:56:59 PM: npm ERR! notsup Valid OS:    darwin
5:56:59 PM: npm ERR! notsup Valid Arch:  any
5:56:59 PM: npm ERR! notsup Actual OS:   linux
5:56:59 PM: npm ERR! notsup Actual Arch: x64
5:56:59 PM: npm ERR! A complete log of this run can be found in:
5:56:59 PM: npm ERR!     /opt/buildhome/.npm/_logs/2022-02-03T01_56_59_145Z-debug.log
5:56:59 PM: Error during NPM install
5:56:59 PM: Build was terminated: Build script returned non-zero exit code: 1
5:56:59 PM: Creating deploy upload records
5:56:59 PM: Failing build: Failed to build site
5:56:59 PM: Failed during stage 'building site': Build script returned non-zero exit code: 1
5:56:59 PM: Finished processing build request in 18.244648393s
```

https://app.netlify.com/sites/eslint/deploys

**Comment by mdjermanovic** [comment]
I updated Build & Deploy > Environment > Environment Variables > NODE_VERSION. Its value was 12, I changed it to 16.

Now deploys use Node v16 and npm v8:

```
11:13:50 AM: Now using node v16.13.2 (npm v8.1.2)
```

It seems to work, as Retry Deploy > Deploy Site was successful.

This was probably caused by a newer version of npm updating package-lock.json in an incompatible way for npm 6, which we were using for deploys.

This issue might be related: https://github.com/npm/cli/issues/3230
